src/pages/en/tutorial/3-components/4.md

```diff
@@ -200,18 +200,45 @@ Those commands are all executed at build time to create static HTML for your sit
 
 ### Test your knowledge
 
-Use these words in the text below. Click the blanks to reveal if you're right!
+1. When does Astro run any JavaScript written in a component's frontmatter?
+  <MultipleChoice>
+    <Option>
+     Astro never runs JavaScript
+    </Option>
+    <Option isCorrect>
+      at build-time
+    </Option>
+    <Option>
+       When a visitor clicks a button
+    </Option>
+  </MultipleChoice>
+
+2. Optionally, Astro can send JavaScript to the browser to allow:
+  <MultipleChoice>
+    <Option>
+      users to click page links
+    </Option>
+    <Option>
+      faster load times
+    </Option>
+    <Option isCorrect>
+       client-side interactivity
+    </Option>
+  </MultipleChoice>
+
+3. Only the JavaScript written or imported here will not be discarded after your site is built:
```

Comment:
“Only… not…” is a confusing formulation. Can we rewrite?

```suggestion
3. The JavaScript written or imported here will be sent to a user’s browser:
```

(Not entirely happy with this, can we do better? 😅)